example/Makefile

```diff
@@ -33,17 +33,22 @@
 # ------------------------------------------------------------------------------
 # Author: J. Henrichs, Bureau of Meteorology
 
```

Comment:
Comment please, e.g. "Makefile driver to test the various examples"

Reply:
Done.